validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -99,8 +101,20 @@ public static ValidatorLoader create(
 
   private Map<BLSPublicKey, Validator> createExternalSignerValidator(
       final ValidatorConfig config, final Supplier<HttpClient> externalSignerHttpClientFactory) {
+    if (config.getValidatorExternalSignerPublicKeys().isEmpty()) {
+      return Collections.emptyMap();
+    }
+
     final Duration timeout = Duration.ofMillis(config.getValidatorExternalSignerTimeout());
 
+    final boolean isReachable =
+        new ExternalSignerUpcheck(
+                externalSignerHttpClientFactory.get(),
+                config.getValidatorExternalSignerUrl(),
+                timeout)
+            .upcheck();
+    STATUS_LOG.externalSignerStatus(config.getValidatorExternalSignerUrl(), isReachable);
```

Comment:
I just wonder if this log message should also be displayed if the signer disappears?

I'm not sure I see anywhere specifically in this latest set of changes, but maybe if the signer stops responding, raising this message would be a useful warning...

Reply:
fixed by upcheck PR